examples/helper/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "helper"
+version = "0.1.0"
+authors = ["Clayton Walker <clayton.m.walker@gmail.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["lib", "cdylib", "staticlib"]
+
+[dependencies]
+png = "0.16"
+libc = {version="0.2", features=[]}
```

Comment:
should this just be `libc = "0.2"`?

Reply:
I was using the same declaration as listed in the root wgpu-native [Cargo.toml](https://github.com/gfx-rs/wgpu-native/blob/master/Cargo.toml#L44). Fixed tho